Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -1178,6 +1178,11 @@ void WindowManager::process_key_event(KeyEvent& event)
         return;
     }
 
+    if (event.type() == Event::KeyDown && event.modifiers() == (Mod_Alt | Mod_Shift) && event.key() == Key_ExclamationPoint) {
+        Compositor::the().invalidate_screen();
+        return;
+    }
+
```

Comment:
Should this shortcut be added inside of the `Kernel: Add VirtIOGPU graphics device` commit? It feels like its own thing that could warrant its own commit.

Reply:
That's a good point. I will move it to its own commit.